js/boltpay/admin_order.js

```diff
@@ -24,6 +25,24 @@ AdminOrder.prototype.prepareParams =
             params.form_key = FORM_KEY;
         }
 
+        ////////////////////////////////////////////////////////////////////
+        // Force billing to populate shipping when 'billing as shipping' is set
+        ////////////////////////////////////////////////////////////////////
+        var billingAddressContainer = $('order-billing_address');
+        if (billingAddressContainer) {
+
+            var isBillingAddressShipping = document.getElementById('order-shipping_as_billing');
+            if (isBillingAddressShipping && isBillingAddressShipping.checked) {
+                var billingAddressData = this.serializeData('order-billing_address');
+                if (billingAddressData) {
+                    billingAddressData.each(function(value) {
+                        document.querySelector("[name='"+value[0].replace('billing_', 'shipping_')+"']").value = value[1];
```

Comment:
suggest checking `document.querySelector("[name='"+value[0].replace('billing_', 'shipping_')+"']")` is non-null before setting value

Reply:
@daisy1754  Updated code to handle this and log it so that anomalies can easily be detected and support extended to these cases.